jobboard/tests/conftest.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+from student.models import Major
+from datetime import date
+from django.contrib.auth.models import User
+from student.models import Student, EducationalInstitution
```

Comment:
While `conftest.py` files are a popular feature of pytest I usually advise teams against using them since they essentially create global implicit dependencies. You end up having a hard time figuring out where the fixtures are coming from.

Its small enough to be acceptable for now, but make sure it stays small!